checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallInference.java

```diff
@@ -493,7 +610,7 @@ private void addOrUpdateClassMustCall() {
   }
 
   /**
-   * Computes ownership transfer at the method call to infer @Owning annotation for the arguments
+   * Computes ownership transfer at the method call to infer @Owning annotations for the arguments
```

Comment:
I am confused by this comment.  Should it be, "Computes ownership transfer at the method call to infer @Owning annotations for formal parameters of the current method, if the parameter is passed into the call and the corresponding formal parameter of the callee is @Owning."?

Reply:
It is correct, I updated the comment to what you suggested.